frontend_tests/puppeteer_tests/recent-topics.ts

```diff
@@ -0,0 +1,33 @@
+import type {Page} from "puppeteer";
+
+import common from "../puppeteer_lib/common";
+
+async function click_mute_and_return_last_topic_id(page: Page): Promise<string | undefined> {
+    return await page.evaluate(() => {
+        const $topic = $('#recent_topics_table tr[id^="recent_topic"]').last();
+        const $mute_icon = $topic.find(".on_hover_topic_mute");
+        $mute_icon.trigger("click");
```

Comment:
We try to avoid JS to trigger click, 
Can you rewrite this function to use `page.click` or `element.clicl`?
And Is there any particular reason for muting the last topic in the list?
If you are using `page.click` then you might have to scroll the page, but If we mute the first topic then we don't have to worry about scrolling.